.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+
+contact_links: 
```

Comment:
Should we add this to issue template only instead of a seperate config.yml, 

also can you explain how this config.yml works with github? haven't explored this @zhravan do you have any idea on this custom config file ?

if possible paste a quick link here for the reference docs to take a look at, 


thanks !